SingularityExecutor/src/main/resources/logrotate.cron.hbs

```diff
@@ -1,6 +1,7 @@
-{{#if logrotateForceHourlyConfig}}
-{{{ cronSchedule }}} root {{{ logrotateCommand }}} -f -s {{{ logrotateStateFile }}} {{{ logrotateForceHourlyConfig }}} {{{ outputRedirect }}}
-{{/if}}
+{{#each logrotateForceConfigs}}
+{{{ cronSchedule }}} root {{{ logrotateCommand }}} -f -s {{{ logrotateStateFile }}} {{{ logrotateForceConfigPath }}} {{{ outputRedirect }}}
+{{/each}}
+
 {{#if logrotateSizeBasedConfig}}
-{{{ cronSchedule }}} root {{{ logrotateCommand }}} -s {{{ logrotateStateFile }}} {{{ logrotateSizeBasedConfig }}} {{{ outputRedirect }}}
+*/5 * * * * root {{{ logrotateCommand }}} -s {{{ logrotateStateFile }}} {{{ logrotateSizeBasedConfig }}} {{{ outputRedirect }}}
```

Comment:
This isn't strictly related to this PR, but I noticed that we were previously only evaluating size-based `logrotateAdditionalFiles` configs every hour. This hardcodes it to every five minutes, which makes it less likely that we let an extremely high-volume logfile fill up a Mesos host. Happy to lengthen this interval (or back out this change entirely) if we think this is too frequent, though.